Examples/Scripts/Optimization/Optuna_tuning.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env python3
+import sys
+
+import sys
+import os
+import yaml
+import pprint
+import time
+import datetime
+import warnings
+
+import optuna
+import logging
+import uproot
+
+import pathlib
+import matplotlib
+matplotlib.use('pdf')
+import matplotlib.pyplot as plt
+import random
+import subprocess
+import multiprocessing
+import numpy as np
+import json
+import array
+import sys
+import argparse
+import pandas as pd
+
+from typing import Optional, Union
+from pathlib import Path
+
+from optuna.visualization import plot_contour
+from optuna.visualization import plot_edf
+from optuna.visualization import plot_intermediate_values
+from optuna.visualization import plot_optimization_history
+from optuna.visualization import plot_parallel_coordinate
+from optuna.visualization import plot_param_importances
+from optuna.visualization import plot_slice
+
+srcDir = Path(__file__).resolve().parent.parent/"Python/"
+
+def run_ckf(params, names, outFile):
```

Comment:
Your really not meant to call the ckf track script that way. You should reimplement runCKFTracks and run it here with that parameters of your choice.